.github/workflows/test-aks.yaml

```diff
@@ -22,6 +22,9 @@ on:
       - edge
   schedule: # Run every 2 hours
     - cron: "0 */2 * * *"
+concurrency:
```

Comment:
Can you add a comment explaining why this is needed? It's easy to forget the rationale for adding something like this later.

Reply:
This is to group workflow runs and cancel if a new workflow starts running. What I mean is that, if we create a PR, a workflow is triggered. Then if we push another commit to that PR, another workflow will be triggered and the in-progress one should be cancelled.


Now that I think about this, because we might have already created resources, I think it would be good to not cancel this workflow and make sure that the resources are deleted if created.

Reply:
What do you think @rynowak?